xdem/coreg/affine.py

```diff
@@ -31,6 +31,7 @@
     _transform_to_bounds_and_res,
     deramping,
 )
+from xdem.dem import DEM
```

Comment:
I was thinking that, eventually, we could wrap the `coreg` class in a `DEM.coregister()` method. Using it here would trigger a circular import issue.

It looks like we can replace `DEM` with `Raster` below. But, eventually, it would also be nice to be able to verify that 3D CRS used during a coregistration are the same, and for this we'd need to use `DEM`... A problem for later, I guess! :sweat_smile:

Reply:
It was lazy to make them DEMs anyway because they don't contain elevation data. Good catch! Fixed in [d471412](https://github.com/GlacioHack/xdem/pull/407/commits/d471412ded75c80548519a840f97ed80eb94e7ca)